client/dashboard/sites/index.tsx

```diff
@@ -76,33 +77,21 @@ export default function Sites() {
 			return;
 		}
 
-		const _defaultView = {
-			...defaultView,
-			...DEFAULT_LAYOUTS[ nextView.type as keyof typeof DEFAULT_LAYOUTS ],
-		};
+		const { updatedViewPreferences, updatedViewSearchParams } = getUpdatedView( {
```

Comment:
`get` was a little confusing to me. Maybe something like `mergeViewPreferences`? Or `mergeViews`?

Reply:
Renamed to `mergeViews` and added comment in f76f5e4b9ca.